cmd/replay/create.go

```diff
@@ -171,7 +171,7 @@ func (c *createCommand) runReplayRequest(jobName, startDate, endDate string) (st
 
 	c.logger.Info("\n> Initiating replay")
 	if c.forceRun {
-		c.logger.Info(logger.ColoredNotice("> Force running replay even if its already in progress"))
+		c.logger.Info("> Force running replay even if its already in progress")
```

Comment:
`c.logger.Warn`. Rule of thumb: use warn (yellow colored) if there's some message that's belongs to important message from user perspective or belongs to soft error (error without interrupt existing execution). Please check the rest sir